packages/pharos-site/src/pages/faqs.mdx

```diff
@@ -0,0 +1,91 @@
+import PageSection from '../../src/components/statics/PageSection.tsx';
+
+<PageSection isHeader title="Pharos FAQ" />
+
+<PageSection title="How can Pharos help my development?" subSectionLevel={2}>
+  <p>
+    Pharos contains visual and behavioral components that are consistent with our design decisions
+    out of the box. Pharos also seeks to overcome the challenges Bonsai faces around framework
+    interoperability, code bloat, and interdependent components.
+  </p>
+</PageSection>
+
+<PageSection
+  title="I tried adopting a component but encountered a number of issues. How do I know when something is usable?"
+  subSectionLevel={2}
+>
+  <p>
+    You can find the status of each component{' '}
+    <PharosLink href="https://pharos.jstor.org/components/component-status">here</PharosLink>. This
+    page will reflect the progress on various aspects of a component's readiness and help you
+    understand what to expect before adopting a component so you can plan accordingly.
+  </p>
+</PageSection>
+
+<PageSection title="What do I do if I still run into issues?" subSectionLevel={2}>
+  <p>
+    Give us the gift of feedback! When you run into issues it is important that you be vocal about
+    it and not let that inhibit or discourage you from adopting them. We need your help to identify
+    the gaps and sincerely welcome feedback no matter how tough it is.
+  </p>
+  <p>
+    The recommended way to provide feedback is via Github issues. Creating an issue in the
+    repository will provide a common place to track and discuss the nature of the issue, as well as
+    potential solutions.
+  </p>
```

Comment:
We could replace this whole paragraph with a short blurb and link to the contribution guidelines.